<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtConceptDialog</class>
 <widget class="QDialog" name="QtConceptDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtExampleDialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <widget class="ribi::cmap::QtExamplesDialog" name="qtexamplesdialog" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_rating_complexity">
     <property name="text">
      <string>Rating complexity</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="box_rating_complexity">
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>2</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
     <property name="displayIntegerBase">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_rating_concreteness">
     <property name="text">
      <string>Rating concreteness</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="box_rating_concreteness">
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>2</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
     <property name="displayIntegerBase">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_rating_specificity">
     <property name="text">
      <string>Rating specificity</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QSpinBox" name="box_rating_specificity">
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>2</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
     <property name="displayIntegerBase">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="box_is_complex">
     <property name="text">
      <string>is complex</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLineEdit" name="edit_name"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_name">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ribi::cmap::QtExamplesDialog</class>
   <extends>QWidget</extends>
   <header>qtconceptmapexamplesdialog.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>box_is_complex</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
